src/sensors/SensorManager.cpp

```diff
@@ -23,12 +23,16 @@
 
 #include "SensorManager.h"
 
+#include <map>
```

Comment:
Map is the slower of the two map variants, while this isn't part of any hot code paths, it also doesn't take much to swap it out to an unordered_map

Reply:
i tried and the constructor complained, so i just think it doesn't matter since it's not used in runtime